include/flatbuffers/reflection.h

```diff
@@ -105,6 +105,20 @@ inline Table *GetFieldT(const Table &table,
   return table.GetPointer<Table *>(field.offset());
 }
 
+// Get a field, if you know it's a struct.
+inline const Struct *GetFieldStruct(const Table &table,
+                                    const reflection::Field &field) {
+  assert(field.type()->base_type() == reflection::Obj);
```

Comment:
The is_struct field is defined in table Object. We can probably get the index from field.type()->index() but the objects vector is defined in the schema table which we don't have in this function. Is there any other way to test if the passed in field is a struct?